{"uid":"bad58b802e8e8350","name":"test_precision[Test for Imbue]","fullName":"tests.test_network_parameters.TestPrecision#test_precision","historyId":"74ffc855de20c1e947196ebc92d12f80","time":{"start":1669869755583,"stop":1669869755632,"duration":49},"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","statusTrace":"self = <tests.test_network_parameters.TestPrecision object at 0x7fc825358730>\nchain = <tests.utils.chain_model.Chain object at 0x7fc82543e730>\n\n    def test_precision(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7fc82543e730>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] Imbue\n\ntests/utils/chain_model.py:29: TimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","statusTrace":"self = <tests.test_network_parameters.TestPrecision object at 0x7fc825358730>\nchain = <tests.utils.chain_model.Chain object at 0x7fc82543e730>\n\n    def test_precision(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7fc82543e730>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] Imbue\n\ntests/utils/chain_model.py:29: TimeoutError","steps":[],"attachments":[{"uid":"519ca17c70698978","name":"stdout","source":"519ca17c70698978.txt","type":"text/plain","size":54}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_network_parameters"},{"name":"subSuite","value":"TestPrecision"},{"name":"host","value":"fv-az196-82"},{"name":"thread","value":"1658-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_network_parameters"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"chain","value":"<tests.utils.chain_model.Chain object at 0x7fc82543e730>"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"dd2eec5cdf97e910","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669869745502,"stop":1669869745564,"duration":62}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":34,"skipped":0,"passed":248,"unknown":0,"total":282},"items":[{"uid":"85d892ef7db1952b","reportUrl":"https://nova-wallet.github.io/test-runner/1998//#testresult/85d892ef7db1952b","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669859317429,"stop":1669859317489,"duration":60}},{"uid":"1a3c7f50f363b4b","reportUrl":"https://nova-wallet.github.io/test-runner/1997//#testresult/1a3c7f50f363b4b","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669840816392,"stop":1669840816445,"duration":53}},{"uid":"3880119c3b12bf1a","reportUrl":"https://nova-wallet.github.io/test-runner/1996//#testresult/3880119c3b12bf1a","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669826724503,"stop":1669826724586,"duration":83}},{"uid":"262469c44ac018c1","reportUrl":"https://nova-wallet.github.io/test-runner/1995//#testresult/262469c44ac018c1","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669812900854,"stop":1669812901067,"duration":213}},{"uid":"854c4e7fd830f7c3","reportUrl":"https://nova-wallet.github.io/test-runner/1994//#testresult/854c4e7fd830f7c3","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669797853487,"stop":1669797853685,"duration":198}},{"uid":"63c5145744cebe15","reportUrl":"https://nova-wallet.github.io/test-runner/1993//#testresult/63c5145744cebe15","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669783374968,"stop":1669783375034,"duration":66}},{"uid":"a7231b8722dff2","reportUrl":"https://nova-wallet.github.io/test-runner/1992//#testresult/a7231b8722dff2","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669772299747,"stop":1669772299879,"duration":132}},{"uid":"680fa289dbde9735","reportUrl":"https://nova-wallet.github.io/test-runner/1991//#testresult/680fa289dbde9735","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669754430914,"stop":1669754431180,"duration":266}},{"uid":"e06c7c1900119587","reportUrl":"https://nova-wallet.github.io/test-runner/1990//#testresult/e06c7c1900119587","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669740374726,"stop":1669740374998,"duration":272}},{"uid":"fc77211fc570ef8","reportUrl":"https://nova-wallet.github.io/test-runner/1989//#testresult/fc77211fc570ef8","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669726391093,"stop":1669726391172,"duration":79}},{"uid":"a07cd9e9ddc5cfe","reportUrl":"https://nova-wallet.github.io/test-runner/1988//#testresult/a07cd9e9ddc5cfe","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669711166350,"stop":1669711166504,"duration":154}},{"uid":"a4aee24fa5fd91e3","reportUrl":"https://nova-wallet.github.io/test-runner/1986//#testresult/a4aee24fa5fd91e3","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669685608240,"stop":1669685608370,"duration":130}},{"uid":"5dcb348ff3cd9ddc","reportUrl":"https://nova-wallet.github.io/test-runner/1985//#testresult/5dcb348ff3cd9ddc","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669667746619,"stop":1669667746692,"duration":73}},{"uid":"15b9142324af60b7","reportUrl":"https://nova-wallet.github.io/test-runner/1984//#testresult/15b9142324af60b7","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669653431912,"stop":1669653431981,"duration":69}},{"uid":"d72a23333d2b742","reportUrl":"https://nova-wallet.github.io/test-runner/1983//#testresult/d72a23333d2b742","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669639776816,"stop":1669639777081,"duration":265}},{"uid":"f77d9348bf633ef2","reportUrl":"https://nova-wallet.github.io/test-runner/1982//#testresult/f77d9348bf633ef2","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669624750095,"stop":1669624750147,"duration":52}},{"uid":"32d182f95f42d2ea","reportUrl":"https://nova-wallet.github.io/test-runner/1981//#testresult/32d182f95f42d2ea","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669610281081,"stop":1669610281139,"duration":58}},{"uid":"912989518e2dc839","reportUrl":"https://nova-wallet.github.io/test-runner/1980//#testresult/912989518e2dc839","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669598999254,"stop":1669598999315,"duration":61}},{"uid":"2c31a351c920b00b","reportUrl":"https://nova-wallet.github.io/test-runner/1979//#testresult/2c31a351c920b00b","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669581299035,"stop":1669581299186,"duration":151}},{"uid":"66753cda138a6735","reportUrl":"https://nova-wallet.github.io/test-runner/1978//#testresult/66753cda138a6735","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","time":{"start":1669567220346,"stop":1669567220589,"duration":243}}]},"tags":[]},"source":"bad58b802e8e8350.json","parameterValues":["<tests.utils.chain_model.Chain object at 0x7fc82543e730>"]}